config/config.go

```diff
@@ -16,5 +16,5 @@ package config
 
 const (
 	OperatorName = "osd-metrics-exporter"
-	OperatorNamespace = "osd-metrics-exporter"
+	OperatorNamespace = "openshift-osd-metrics"
```

Comment:
Is this an unrelated change?

Reply:
Yeah. But it was broken and it's related to the cleanup. I've mentioned it in the PR description.